Dockerfile

```diff
@@ -32,7 +34,6 @@ FROM debian:buster-slim
 RUN apt-get update
 RUN apt-get install -y ca-certificates
 
-RUN mkdir /app
```

Comment:
Removed these as `WORKDIR` creates the folder if it doesn't exist: https://docs.docker.com/engine/reference/builder/#workdir

Reply:
thanks, I didn't know that :+1: